### DISCUSSION TITLE
REPCA1 control scheme

### DISCUSSION THREAD

**Discussion by jinningwang** [CURENT/andes#278] [Modeling] [answered]
Hi Hantao,

In the most recent [REPCA1](https://github.com/cuihantao/andes/blob/b8b9212c0d2fa24f6af81e1fca3df1034470eea4/andes/models/renewable/repca1.py) model, it seems does not export any control variables to other models. Is it a TODO item we haven't implemented?

Regards,
Jinning

**Comment by cuihantao** [comment, marked as answer]
It’s in line 488 and 490

**Reply by jinningwang** [reply]
Thanks for your reply. I missed those two lines.
